examples/tycho-surefire/tycho-surefire-example-test/pom.xml

```diff
@@ -0,0 +1,108 @@
+	            <exclusion>
+		            <groupId>org.ops4j.pax.exam</groupId>
+		            <artifactId>pax-exam</artifactId>
+	            </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+    
+    <build>
+        <plugins>  
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.10</version>
+                
+                <!-- 
+                Copy dependencies to a lib directory so they can be included on 
+                the classpath in the build runtime 
+                -->
+                <executions>
+                    <execution>
+                        <id>copy-dependencies</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <excludeTransitive>false</excludeTransitive>
+                            <excludeGroupIds>p2.eclipse-plugin</excludeGroupIds>
+                            <includeScope>runtime</includeScope>
+                            <includeTypes>jar</includeTypes>
+                            <outputDirectory>lib</outputDirectory>
+                            <overWriteReleases>true</overWriteReleases>
+                            <overWriteSnapshots>true</overWriteSnapshots>
+                            <overWriteIfNewer>true</overWriteIfNewer>
+                            <stripVersion>true</stripVersion>
+                        </configuration>
+                    </execution>
+                    
+                    <!--
+                    In order for the Cucumber OsgiClassFinder to find the 
+                    Cucumber annotation classes, they must be explicitly listed 
+                    as classes in the OSGi bundle.  So, they're being 
+                    extracted from the cucumber jar and included in the 
+                    plugin test jar file 
+                    -->
+                    <execution>
+                        <id>unpack-dependencies</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>io.cucumber</groupId>
+                                    <artifactId>cucumber-java</artifactId>
+                                    <version>${project.version}</version>
+                                    <outputDirectory>${project.build.directory}/classes</outputDirectory>
+                                    <includes>cucumber/api/java/Before.class, cucumber/api/java/After.class, cucumber/api/java/en/*.class</includes>
```

Comment:
Do all end users have to do this? That would kinda suck.

Reply:
Yes, all end users have to do this, and yes, it kind of sucks.  My example is only extracting the English ones because the example feature file is written in English.

I'll add comments to the file explaining about needing to extract the classes for the actual language of the feature files.

Reply:
Yes, all end users have to do this, and yes, it kind of sucks.  My example is only extracting the English ones because the example feature file is written in English.

I'll add comments to the file explaining about needing to extract the classes for the actual language of the feature files.